ntoskrnl/mm/ARM3/sysldr.c

```diff
@@ -2873,7 +2874,7 @@ LdrpFetchAddressOfSecurityCookie(PVOID BaseAddress, ULONG SizeOfImage)
     /* Check for sanity */
     if (!ConfigDir ||
         DirSize < FIELD_OFFSET(IMAGE_LOAD_CONFIG_DIRECTORY, SEHandlerTable) ||  /* SEHandlerTable is after SecurityCookie */
-        (ConfigDir->Size != DirSize))
+        (ConfigDir->Size < sizeof(IMAGE_LOAD_CONFIG_DIRECTORY)))
```

Comment:
And if the configdir size is larger, it's OK?

Reply:
Not allowing larger size could make it ignore newer images with updated `IMAGE_LOAD_CONFIG_DIRECTORY`. Actually now that I think about it now, there could be a problem too with loading older images in long term when ROS gets updated struct, some maybe I should just use magic value like here: https://github.com/reactos/reactos/blob/88107adc9249bcc7eb691a8b3f97a797bb58d5ab/dll/ntdll/ldr/ldrinit.c#L434
As to why changing this in first place, `ConfigDir->Size` and `DirSize` seems to be always different (72 vs. 64).